packages/indexer/src/indexers/autocrat/autocrat-proposal-indexer.ts

```diff
@@ -35,48 +37,29 @@ export enum AutocratDaoIndexerError {
   NothingToInsertError = "NothingToInsertError",
 }
 
-//TODO we want to make an RPC call for the block time and use that as the createdAt
-// then cascade that by using slots per proposal times 4000 ms
 export const AutocratProposalIndexer: IntervalFetchIndexer = {
   cronExpression: "30 * * * * *",
   index: async () => {
+
     try {
+      let currentSlot = (
```

Comment:
You can actually grab created_at from this and it's going to be within 1-5 second of the actual time for the slot, so no need for RPC call.